packages/math/src/pool/index.ts

```diff
@@ -1,2 +1,5 @@
 export * from "./estimates";
-export * from "./weighted";
+export * from "./general";
+export * from "./multihop";
+export { StableSwapToken, StableSwapMath } from "./stable";
+export { WeightedPoolMath } from "./weighted";
```

Comment:
I did this because there are name conflicts with functions in stable and weighted, but I didn't want to rename them. Also, I noticed that wherever name functions were imported, they already used `import * as Math` so I follow the same patter except I export the wrapped object directly.

Reply:
OK. make sense.